README.md

```diff
@@ -1,12 +1,12 @@
 # pwa-prototype-reporting
-A playground for a PWA for reporting only purposes. This project started as an experimental prototype to migrate the functionality of Ushahidi survey submissions into a small standalone project that can be deployed anywhere and in the future could be linked to any ushahidi.io deployment chosen by the user.
+ This project started as an experimental prototype to migrate the functionality of Ushahidi survey submissions into a small standalone project that can be deployed anywhere and in the future could be linked to any ushahidi.io deployment chosen by the user.
 
 # Offline support
 - Supports loading the form while offline if you have used the app in the past
 - Saving form responses while offline and syncing the response with the backend when the user is online
 
 # Dynamic attribute loading
-- Loads the deployment attributes for any ushahidi v3 form
+- [Pending] Loads the deployment attributes for any ushahidi v3 form
```

Comment:
nice. 
When you finish the offline sync and the form id thing, you can remove the [pending] tags and I'll merge everything together so it's less confusing :) thank you for all your hard work on this 🎉